File: 1000/CF958B1_Maximum_Control_easy.java
import java.util.Scanner;

public class CF958B1_Maximum_Control_easy
{
    public static void main(String[] args) 
    {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int[] planets=new int[n+1];
        int remotePlanets=0;
        for (int i=0;i<(n-1)*2;i++)
        {
            int planet=sc.nextInt();
            planets[planet]++;
        }
        for (int i:planets)
        {
            if (i==1)
            {
                remotePlanets++;
            }
        }
        System.out.println(remotePlanets);
        sc.close();
    }
}

File: 1000/CF958B1_Maximum_Control_easy.py
import sys

data = list(map(int, sys.stdin.read().split()))

n = data[0]
planets = [0] * (n + 1)
remotePlanets = 0

for planet in data[1:]:
    planets[planet] += 1

for i in planets:
    if i == 1:
        remotePlanets += 1

print(remotePlanets)